src/scene/materials/standard-material.js

```diff
@@ -1094,7 +1147,7 @@ function _defineMaterialProps() {
     _defineColor('attenuation', new Color(1, 1, 1));
     _defineFloat('emissiveIntensity', 1);
     _defineFloat('specularityFactor', 1);
-    _defineFloat('sheenGlossiness', 0);
+    _defineFloat('sheenGloss', 0);
```

Comment:
so this renames the property .. what would happen to materials people have set up with the old name?

Search the codebase for RENAMED_PROPERTIES .. I wonder if that could handle it? (I have not used it before, but worth a test)

Reply:
This parameter wasn't exposed in the editor before, so I doubt anyone has used it. But I can definitely add it to the renamed properties list just to make sure.

Reply:
ah true, ignore in that case, there's no way the parameter would be stored in the database then.

Reply:
people could have used it in the engine scripts is the only place .. do we have some way to print warning on setting an incorrect parameter or something?

Reply:
Hmm, if it's not called that in the standard-material-parameters, it will be shown in the log as an invalid standard material parameter. But that's about it afaik.

Reply:
great, nothing to do here, resolving.